docs/specs/casing.yml

```diff
@@ -29,7 +29,7 @@ inputs:
     [redirect](REDIRECT.md)
 outputs:
   docs/a.json: |
-    {"content":"<p><a href=\"/path\">redirect</a></p>"}
+    {"content":"<p><a href=\"redirect\">redirect</a></p>"}
```

Comment:
I removed a non-legacy behavior to simplify our test metric.

Reply:
is it worth to change our prod behavior just for simplifing our test metric?

Reply:
> is it worth to change our prod behavior just for simplifing our test metric?

Our prod is using `legacy` behavior, the one on the right side.